2024-02-09T08:53:46.927932+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '0e81569d-7677-497f-bbc8-845c79b3bc87', 'counter': 1, 'stdout': '\x1b[0;31mERROR! the playbook: playbook3.yaml could not be found\x1b[0m', 'start_line': 0, 'end_line': 1, 'runner_ident': 'a62e5d65-ea41-494d-8431-84a3a6884709', 'created': '2024-02-09T08:53:46.918011'}
2024-02-09T08:53:47.215044+00:00 ERROR 'ansible_navigator.actions.run_playbook.write_artifact' Saving the artifact file failed, resulted in the following error: f[Errno 13] Permission denied: '/playbook3-artifact-2024-02-09T08:53:47.212554+00:00.json'
